# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoderStateMachine.h
# Commit: 3e647187e78b
# Full Hash: 3e647187e78b9f74a372031572e4b0a502f72658
# Author: alwu <alwu@mozilla.com>
# Date: 2022-11-05 21:53:50
# Regressor Bug: 1262276
# File Overlap Count: 1
# Description:
#   Bug 1262276 - part2 : support seamless video looping. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159126
# ==============================================================================

diff -r 1df4774b2c53 -r 3e647187e78b dom/media/MediaDecoderStateMachine.h
--- a/dom/media/MediaDecoderStateMachine.h	Sat Nov 05 02:23:28 2022 +0000
+++ b/dom/media/MediaDecoderStateMachine.h	Sat Nov 05 02:23:29 2022 +0000
@@ -512,6 +512,7 @@
   // to adjust sample time from clock time to media time.
   void AdjustByLooping(media::TimeUnit& aTime) const;
   Maybe<media::TimeUnit> mAudioDecodedDuration;
+  Maybe<media::TimeUnit> mVideoDecodedDuration;
 
   // Current playback position in the stream in bytes.
   int64_t mPlaybackOffset = 0;
